tests/preprocessing/test_imputation.py

```diff
@@ -46,6 +48,11 @@ def _base_check_imputation(
     Raises:
         AssertionError: If any of the checks fail.
     """
+    # if .x of the AnnData is a dask array, convert it to a numpy array
```

Comment:
```suggestion
    # if .X of the AnnData is a dask array, convert it to a numpy array
```

Reply:
But I can see this in the code. Can you write WHY you're doing this, please? Much more useful.